xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/breadcrumbs.less

```diff
@@ -52,3 +52,9 @@
     }
   }
 }
+
+dd .location-actions {
+  /* Retrieve style from the breadcrumb block. Needed for keyboard accessibility reorganization of the
+  location-picker component. */
+  .breadcrumb;
```

Comment:
The best would be to have this inside ``locationPicker.css`` but we don't support LESS there ATM so we're left with two options:

* keep this (temporarily) inside ``breadcrumb.less``
* update the JavaScript code to use a more specific CSS selector when updating the actual breadcrumb.

I'm not sure what's best. I guess the first option is acceptable for now.